requirements.txt

```diff
@@ -67,8 +67,6 @@ quark-engine==20.12
 git+https://github.com/DissectMalware/XLMMacroDeobfuscator.git@89fbce0c87014a4b5a22c1aef09c8b3ea9bf16c0
 speakeasy-emulator==1.4.8
 sphinx_rtd_theme==0.5.0
-flake8==3.8.4
-black==20.8b1
 pre-commit==2.9.2
```

Comment:
sphinx_rtd_theme and pre-commit are also part of testing/development

Reply:
You don't have a `.readthedocs.yml` file. 
If i move the sphinx import, the readthedocs should not build anymore.